index.bs

```diff
@@ -735,12 +735,13 @@ Any time a new [=sensor reading=] for a [=platform sensor=] is obtained and if t
 [=can expose sensor readings=], [=update latest reading=] is invoked with the [=platform sensor=]
 and the [=sensor reading=] as arguments.
 
-The [=latest reading=] [=ordered map|map=]
-contains an [=map/entry=]
-whose [=map/key=] is "timestamp" and
-whose [=map/value=] is a high resolution timestamp of the time
-at which the [=latest reading=] was obtained
-expressed in milliseconds that passed since the [=time origin=].
+The [=latest reading=] [=ordered map|map=] contains an [=map/entry=] whose [=map/key=] is
+"timestamp" and whose [=map/value=] is a high resolution timestamp estimating the
```

Comment:
s/estimating the/that represents the estimate of the/

Reply:
Or *timestamp that estimates the*